subprojects/docs/src/samples/userguide/files/archivesWithJavaPlugin/groovy/build.gradle

```diff
@@ -13,12 +13,18 @@ dependencies {
     implementation 'commons-io:commons-io:2.6'
 }
 
+task configureUberJar {
+    doLast {
+        // Don't do this during configuration phase!
+        uberJar.from(configurations.runtimeClasspath.findAll { it.name.endsWith('jar') }.collect { zipTree(it) })
```

Comment:
As we discussed let's leave this part as it is because this is a single project without any project dependencies so the sample actually works fine. If people copy/paste this sample to a multiproject, the build would fail.

Reply:
Why a separate task here? You can simply wrap the from argument in a Closure.

`from { configurations... }`

Reply:
We wanted to have a consistent solution which works for this sample and the Javadoc one. We can also use `from { configurations }` here if you prefer.

Reply:
I would much prefer that, yes.